docs/functions.rst

```diff
@@ -1285,7 +1285,7 @@ Arguments:
    :align: left
    :widths: 5,5,5,15
 
-   `name`, Yes, `string`, name of the VolumeSnapshot
+   `name`, No, `string`, name of the VolumeSnapshot. Default value is ``<pvc>-snapshot-<30-char-random-alphanumeric-suffix>``
```

Comment:
Like mentioned at other place 30 is a bit too much. If this is how k8s does recommends is then its ok.

Reply:
K8s has a limit of 253 chars